snuba/migrations/clickhouse.py

```diff
@@ -1,4 +1,5 @@
-CLICKHOUSE_SERVER_MIN_VERSION = "21.8.12.29"
-# Note: 21.8.12.29 and 21.8.13.1 are used in self-hosted builds
-# even though SaaS clusters are all on 22.8 or above
-CLICKHOUSE_SERVER_MAX_VERSION = "23.8.11.29"
+CLICKHOUSE_SERVER_MIN_VERSION = "22.8.15.25"
+# Note: some test regions are using 22.8.15.25.
```

Comment:
which ones?

Reply:
@volokluev I linked them in the PR description, I'm not sure if they are still in use.

Reply:
Should be none by now
```
/Users/dfedorov/github/ops/salt/pillar/clickhouse
➜  clickhouse git:(master) grep -R 22.8 *
➜  clickhouse git:(master) 

```